app/controllers/collections_controller.rb

```diff
@@ -54,6 +54,16 @@ def new
     end
   end
 
+  def update
+    @collection = Collection.find(params[:id])
+
+    if @collection.update(create_collection_params)
```

Comment:
```suggestion
    if @collection.update(update_collection_params)
```
Let's make sure a user can't accidentally update the version, or scenarios when the update form does not yet contain them. We'd need something like `update_collection_params`.